src/callback.rs

```diff
@@ -35,6 +35,30 @@ impl PyCallbackOutput for () {
     const ERR_VALUE: Self = ();
 }
 
+#[doc(hidden)]
+pub trait IntoResult<R> {
+    fn into_result(self) -> R;
+}
+
+impl<T> IntoResult<PyResult<T>> for T
+where
+    T: IntoPy<PyObject>,
+{
+    fn into_result(self) -> PyResult<Self> {
```

Comment:
```suggestion
    #[inline]
    fn into_result(self) -> PyResult<Self> {
```

Reply:
Does this even do anything on a generic function? Or on a trait implementation?

Reply:
Hmmm good question. Looking at https://matklad.github.io/2021/07/09/inline-in-rust.html

.... I think because this trait is generic this is necessarily a generic function, so implicitly inlineable anyway?